special-pages/pages/duckplayer/app/components/MobileApp.module.css

```diff
@@ -87,6 +87,12 @@ body:has([data-state="completed"] [aria-checked="true"]) .switch {
     padding-bottom: 0;
     border-top-left-radius: var(--outer-radius);
     border-top-right-radius: var(--outer-radius);
+
```

Comment:
This fixes a subtle animation issue that becomes more visible when there’s higher contrast between backgrounds.

(first example needs a massive zoom in)

<img width="771" alt="image" src="https://github.com/user-attachments/assets/f4b51929-155a-4b44-a4db-0741f32ecf2b" />

<img width="578" alt="image" src="https://github.com/user-attachments/assets/d477ee1e-d1fd-448a-a50a-23ee28722a60" />